test/formtools.js

```diff
@@ -560,23 +560,6 @@ describe('formtools', function () {
                             label: 'Custom action'
                         }
                     ],
-                    summary: {
```

Comment:
@smebberson The unit tests was created prior to vagrant setup. We need an environment to run the tests. Also the tests are over 2 years old so there are a lot of breaking changes included since, so we'll need to set aside time to get this up and running again.